examples/js/exporters/GLTFExporter.js

```diff
@@ -926,6 +926,15 @@ THREE.GLTFExporter.prototype = {
 				var attribute = geometry.attributes[ attributeName ];
 				attributeName = nameConversion[ attributeName ] || attributeName.toUpperCase();
 
+				// JOINTS_0 must be UNSIGNED_BYTE or UNSIGNED_SHORT.
+				if ( attributeName === 'JOINTS_0' &&
+						! ( attribute instanceof THREE.Uint16BufferAttribute ) &&
+						! ( attribute instanceof THREE.Uint8BufferAttribute ) ) {
+
+					attribute = new THREE.Uint16BufferAttribute( new Uint16Array( attribute.array ), attribute.itemSize, attribute.normalized );
```

Comment:
I think just in case we need warning here if we down size from `Uint32` or `Float32` because data can be broken if it has values over 0xFFFF (probably it's very rare tho).

Reply:
Is there a max range for indices in WebGL? If not, is there any reason to prefer Uint16 over Uint32 here (besides memory)?

Reply:
> Is there a max range for indices in WebGL?

Three.js handles skinning in GPU. Bone Matrices can be texture or uniforms array in shader. They can limit the bone indices range. So I know over 0xFFFF is too huge.

> is there any reason to prefer Uint16 over Uint32 here (besides memory)?

IMO, no. But we decided to aggressively check/verify parameters in exporter. So this checking and display warning is good to me.